### PR TITLE
Update to analytics section in /store

### PR DESCRIPTION
**Pull request by Lukewh**
To keep jujucharms.com/store and GUI store consistent with changes requested by Kevin Monroe.

Also for context - these changes are to point to more up to date charms.

### CODE REVIEW

**Comment by Lukewh** [comment]
*QA*
Within 'Analytics' section of the store:

- Apache analytics SQL > Hive
- Apache Spark > Spark
- Apache Zeppelin > Zeppelin

File: jujugui/static/gui/src/app/components/store/store.js [hunk 1]
```diff
@@ -637,70 +637,54 @@ YUI.add('store', function() {
...
-                                <img src={this._generateIconPath('mariadb')}
-                                    alt=""
-                                    className="featured-entity__image" />
-                            </li>
-                            <li className="featured-entity__image-list-item">
-                                +3
-                            </li>
-                        </ul>
+                        <div className="one-col no-margin-bottom">
+                          <img src={this._generateIconPath(
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/store/store.js]: Unnecessary newline: `this._generateIconPath('hive')`.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7024/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7025/

**Comment by Lukewh** [comment]
:shipit:

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**